### ISSUE TITLE
RuntimeError: no validator found for <class 'starlette.responses.Response'>

### ISSUE THREAD

**Issue by nrshapiro** [tiangolo/fastapi#358]
**Describe the bug**
This example from the documentation does not work
```
from fastapi import FastAPI
from starlette.responses import Response

app = FastAPI()


@app.get("/headers-and-object/")
def get_headers(response: Response):
    response.headers["X-Cat-Dog"] = "alone in the world"
    return {"message": "Hello World"}
```

**To Reproduce**
1. Try the example under "Use a Response parameter"
2. See error

**Expected behavior**
It should run and add to the response.  It gives an error instead:
    builtins.RuntimeError: no validator found for <class 'starlette.responses.Response'>

**Environment:**
 - OS: Windows 
 - FastAPI Version [e.g. 0.3.0], get it with: 27.2

- Python version, get it with: 3.7.3

**Additional context**
There's also no validator for JSONResponse, but that's not needed because it's not required to be a parameter of the function.

**Comment by jekirl** [comment]
Looks to be working with `0.31.0`....Was that example in the docs for `0.27.2`?

See below:
```from fastapi import FastAPI
from starlette.responses import Response
from starlette.testclient import TestClient

app = FastAPI()


@app.get("/headers-and-object/")
def get_headers(response: Response):
    response.headers["X-Cat-Dog"] = "alone in the world"
    return {"message": "Hello World"}

def test_bug():
    client = TestClient(app)
    r = client.get("/headers-and-object/")
    assert r.headers["X-Cat-Dog"] == "alone in the world"
    assert r.json() == {"message": "Hello World"}```

**Comment by nrshapiro** [comment]
Yes, it was fixed in a later version than I had--which Tiangolo pointed 
out in the gitter chat.  I hadn't installed it that long ago and it 
never occured to me that the version could be out-of-date.

And I was using the online docs, so probably were in a later version 
anyway.

All better. :)

Regards--Neil

------ Original Message ------

**Comment by jonathanunderwood** [comment]
I think this can be closed, right?

**Comment by tiangolo** [comment]
Thanks for the help @jekirl and @jonathanunderwood ! :cake: 

I'm glad you solved your problem @nrshapiro , I'll close this now. :tada:
